src/o365/graph/commands/reports/reports-teams-device-usage-user-detail.ts

```diff
@@ -0,0 +1,153 @@
+import auth from '../../GraphAuth';
+import config from '../../../../config';
+import commands from '../../commands';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate
+} from '../../../../Command';
+import { GraphItemsListCommand } from '../GraphItemsListCommand';
+import request from '../../../../request';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+const maxDays = 28;
+const maxDate: Date = new Date();
+maxDate.setDate(maxDate.getDate() - maxDays);
```

Comment:
Since maxDays and maxDate are specific to the command, they should be defined inside the class.

Reply:
Fixed and ready for review.